tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -68,37 +69,45 @@ Verify GPU Operator Deployment  # robocop: disable
     # ...   nvidia-operator-validator DS
 
 Verify That Prometheus Image Is A CPaaS Built Image
-    [Tags]    Sanity   
+    [Tags]    Sanity
     ...     Tier1
-    ...     ODS-734    
+    ...     ODS-734
     ${pod} =    Search Pod    namespace=redhat-ods-monitoring    pod_start_with=prometheus-
     Verify Container Image    redhat-ods-monitoring    ${pod}    prometheus
     ...    "registry.redhat.io/openshift4/ose-prometheus"
     Verify Container Image    redhat-ods-monitoring    ${pod}    oauth-proxy
     ...    "registry.redhat.io/openshift4/ose-oauth-proxy:v4.8"
 
 Verify That Grafana Image Is A Red Hat Built Image
-    [Tags]    Sanity    
+    [Tags]    Sanity
     ...     Tier1
-    ...     ODS-736    
+    ...     ODS-736
     ${pod} =    Search Pod    namespace=redhat-ods-monitoring    pod_start_with=grafana-
     Verify Container Image    redhat-ods-monitoring    ${pod}    grafana
     ...    "registry.redhat.io/rhel8/grafana:7"
     Verify Container Image    redhat-ods-monitoring    ${pod}    auth-proxy
     ...    "registry.redhat.io/openshift4/ose-oauth-proxy:v4.8"
 
 Verify That Blackbox-exporter Image Is A CPaaS Built Image
-    [Tags]    Sanity    
+    [Tags]    Sanity
     ...     Tier1
-    ...     ODS-735    
+    ...     ODS-735
     ${pod} =    Search Pod    namespace=redhat-ods-monitoring    pod_start_with=blackbox-exporter-
     Verify Container Image    redhat-ods-monitoring    ${pod}    blackbox-exporter
     ...    "quay.io/integreatly/prometheus-blackbox-exporter:v0.19.0"
 
 Verify That Alert Manager Image Is A CPaaS Built Image
-    [Tags]    Sanity    
+    [Tags]    Sanity
     ...     Tier1
-    ...     ODS-733    
+    ...     ODS-733
     ${pod} =    Search Pod    namespace=redhat-ods-monitoring    pod_start_with=prometheus-
     Verify Container Image    redhat-ods-monitoring    ${pod}    alertmanager
     ...    "registry.redhat.io/openshift4/ose-prometheus-alertmanager"
+
+Verify Versioning Each Release
+    [Documentation]    Verify versioning each release
+    [Tags]    Sanity
+    ...       ODS-478
+    @{list} =    Oc Get    kind=ClusterServiceVersion    label_selector=olm.copiedFrom=redhat-ods-operator
```

Comment:
wasn't possible to use `Get RHODS version` which is already implemented?

Reply:
just checked and that keyword fetch only the x.y.z. However, i feel like we should have a common keyword to get the rhods version. You may change the already implemented one to retrieve also the full version format (x.y.z-build). Maybe by using an argument to pilot the keyword to return the full or partial version.

Reply:
I think the best approach would be to have `Get RHODS Version` return the full version, but then you'd also need to modify the keyword used for the version comparison to modify the returned value and drop the `-build`

Reply:
why drop -build , because in the description it is written we need to check complete version. https://issues.redhat.com/browse/RHODS-1695

Reply:
What I mean is that if you modify `Get RHODS version` (https://github.com/red-hat-data-services/ods-ci/blob/master/tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot#L93) to return the full version, then you'd also need to modify `Is RHODS Version Greater Or Equal Than` (https://github.com/red-hat-data-services/ods-ci/blob/master/tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot#L102) to remove the `-build` in there.

So you'd have that `Get RHODS version` becomes:
```
Get RHODS version
    #@{list} =  OpenShiftCLI.Get  kind=ClusterServiceVersion  label_selector=olm.copiedFrom=redhat-ods-operator
    #&{dict} =  Set Variable  ${list}[0]
    #Log  ${dict.spec.version}
    ${ver} =  Run  oc get csv -n redhat-ods-operator | grep "rhods-operator" | awk '{print $1}' | sed 's/rhods-operator.//'
    [Return]  ${ver}
```

And `Is RHODS Version Greater Or Equal Than` becomes:
```
Is RHODS Version Greater Or Equal Than
    [Arguments]  ${target}
    ${ver} =  Get RHODS version
    ${ver} =  Fetch From Left  ${ver}  -
    ${comparison} =  GTE  ${ver}  ${target}
    [Return]  ${comparison}
```

Reply:
I agree with @lugi0

Reply:
Updated.